trade-finance-manager-ui/server/controllers/case/amendments/amendmentIsUsingFacilityEndDate.get.test.js

```diff
@@ -0,0 +1,124 @@
+import { AMENDMENT_STATUS, FACILITY_TYPE, MAPPED_FACILITY_TYPE, TEAM_IDS } from '@ukef/dtfs2-common';
+import api from '../../../api';
+import { mockRes } from '../../../test-mocks';
+import { MOCK_AMENDMENT_COVERENDDATE_CHANGE, MOCK_AMENDMENT_FACILITYVALUE_CHANGE } from '../../../test-mocks/amendment-test-mocks';
```

Comment:
Has `MOCK_AMENDMENT_FACILITYVALUE_CHANGE ` been there for a while? if new can it be changed to `MOCK_AMENDMENT_FACILITY_VALUE_CHANGE `, else leave it as is